COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -224,31 +234,47 @@ Follow these steps:
 
 2. Click on the pencil icon to edit your profile. A dialog box opens displaying the current information in your profile, which you can edit as required.
```

Comment:
There is no longer a pencil icon involved. So this line will need changes in the description :)